test/draw/gpu/GpuImageRotationTest.ooc

```diff
@@ -45,6 +45,7 @@ GpuImageRotationTest: class extends Fixture {
 		this add("GPU rotation small X (BGRA)", func {
 			correctImage := RasterBgra open("test/draw/gpu/correct/rotation_small_bgra_X.png")
 			gpuImage := gpuContext createBgra(sourceImage size)
+			gpuImage canvas pen = Pen new(ColorBgra new())
```

Comment:
Maybe because Canvas defaults to `Pen new() // no alpha` which now defaults to `Pen new(ColorBgra new(0, 0, 0, 255)) // alpha == 1.0` ?